### ISSUE TITLE
[BUG] Remote Harvester debug output: [The semaphore timeout period has expired]

### ISSUE THREAD

**Issue by spencerhite** [Chia-Network/chia-blockchain#7681]
I try to start my remote harvester on different IP and am faced with <ssl.SSLContext object at 0x00000291AA3FF340> [The semaphore timeout period has expired].

The correct external IP is in the config file, I’ve tried using both 1.2.1 and 1.2.2 with fresh installs. I’ve reinstalled network adapters, taken firewall down to test, used CHKDSK, made sure I’m using the correct CA from full node. Nothing works and I’m always given the same debug output

**To Reproduce**

1. Go to powershell
2. Go to chia main net directory
3. Start harvester ONLY
4. Debug log shows the problem
**Expected behavior**
“Connected to host: xxxxxxxx:xxxx”

**Desktop**

- OS: Windows 10
- OS Version/Flavor: 21H2
- CPU: AMD Ryan 3950x

Snippet of Debug log

```
2021-07-20T17:23:58.696 harvester harvester               : INFO     Started harvester service on network_id: mainnet
2021-07-20T17:23:58.697 harvester harvester               : INFO     Reconnecting to peer {'host': 'xxxxxxxxx', 'port': 8447}
2021-07-20T17:23:58.707 daemon chia.daemon.server         : INFO     Register service {'service': 'chia_harvester'}
2021-07-20T17:23:58.708 daemon chia.daemon.server         : INFO     registered for service chia_harvester
2021-07-20T17:23:58.708 daemon chia.daemon.server         : INFO     {'success': True}
2021-07-20T17:24:19.730 harvester harvester_server        : INFO     ### Cannot connect to host xxxxxxxx:8447 ssl:<ssl.SSLContext object at 0x00000291AA3FF340> [The semaphore timeout period has expired]
2021-07-20T17:24:22.732 harvester harvester               : INFO     Reconnecting to peer {'host': 'xxxxxxx', 'port': 8447}->

```

**Comment by lpenrod** [comment]
Same problem, different Intel CPU.

**Comment by CuppaChia** [comment]
Did anyone find a solution to this problem, i'm getting it too. Cannot connect to host xx.xxx.xxx.xxx:8444 ssl:<ssl.SSLContext object at 0x0000021697D16D40> [The semaphore timeout period has expired]

**Comment by 85256638** [comment]
i started to have the same issue again.

**Comment by GutRaz** [comment]
Same issue after updating to 2.2.0 and returning to 2.1.4

2024-03-01T17:39:26.525 wallet wallet_server              : INFO     Connection closed: 73.16.241.210, node id: 4d4d826a44e4a5859de7861f9dca82975a44e9a4ecd5d39047ba3b26faa3dc7a
2024-03-01T17:39:26.525 wallet chia.wallet.wallet_node    : INFO     Connected peer PeerInfo(_ip=IPv4Address('73.16.241.210'), _port=8444) is trusted: False
2024-03-01T17:39:26.526 wallet wallet_server              : INFO     Connected with full_node PeerInfo(_ip=IPv4Address('73.16.241.210'), _port=8444)
2024-03-01T17:39:26.751 wallet wallet_server              : INFO     Cannot connect to host 178.131.97.231:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:30.024 wallet wallet_server              : INFO     Cannot connect to host 2.31.52.220:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:30.059 full_node full_node_server        : INFO     Connection closed: 77.21.112.17, node id: 9029e2c0ee41235eafb50438a768171e0f7b737832836ed19b6b231b518936a8
2024-03-01T17:39:30.059 full_node chia.full_node.full_node: INFO     peer disconnected PeerInfo(_ip=IPv4Address('77.21.112.17'), _port=8444)
2024-03-01T17:39:30.793 wallet wallet_server              : INFO     Cannot connect to host 96.255.248.61:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:31.552 wallet wallet_server              : INFO     Cannot connect to host 79.116.134.197:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:31.702 full_node chia.full_node.full_node: INFO      Updated peak to height 5017692, weight 15912709200, hh 2f4018f9c9ca21f3ae685934ebec45ebf84fca8f3f951303704a3c2a87f52e67, forked at 5017691, rh: 8dbf6090bd1aa87e1618f4bc0d19bc1a4cb0f5c75b33f13a0d662f64c9e9d1fb, total iters: 29313234218260, overflow: True, deficit: 16, difficulty: 12160, sub slot iters: 583008256, Generator size: No tx, Generator ref list size: No tx
2024-03-01T17:39:31.713 wallet chia.wallet.wallet_state_manager: INFO     set_sync_mode syncing - range: 5017691-5017692
2024-03-01T17:39:31.714 wallet chia.wallet.wallet_blockchain: INFO     Peak set to: 5017692 timestamp: 1709307508
2024-03-01T17:39:32.319 wallet wallet_server              : INFO     Cannot connect to host 31.148.8.86:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:32.759 wallet wallet_server              : INFO     Cannot connect to host 2403:ad80:a7:2::2:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The network location cannot be reached. For information about network troubleshooting, see Windows Help]
2024-03-01T17:39:33.088 wallet wallet_server              : INFO     Cannot connect to host 62.197.154.37:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:33.479 full_node chia.full_node.full_node: INFO       Finished signage point 1/64: CC: 04c353cc7309e519c455d061398fbdac40fafd5a91af09f93e134b73fa89726b RC: 61d41dd6037b877d30cef63e218d898e7ebfb47c3fa29e06d479e67bd832bce7
2024-03-01T17:39:33.493 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming e14f72bfd8... Found 0 proofs. Time: 0.00637 s. Total 1278 plots
2024-03-01T17:39:33.860 wallet wallet_server              : INFO     Cannot connect to host 89.65.139.82:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:34.408 full_node full_node_server        : INFO     Connection closed: 83.38.90.227, node id: 83e18e61316c61f8992b99f39acd4cde8d3175b974a13b0fca22317f3a04c8aa
2024-03-01T17:39:34.408 full_node chia.full_node.full_node: INFO     peer disconnected PeerInfo(_ip=IPv4Address('83.38.90.227'), _port=0)
2024-03-01T17:39:34.631 wallet wallet_server              : INFO     Cannot connect to host 47.55.14.96:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:35.156 full_node full_node_server        : INFO     Connection closed: 85.254.75.173, node id: 2ee118137cadbfba4f572a0d8627407e554e1b27f75ec1e84997cea5264a8bef
2024-03-01T17:39:35.156 full_node chia.full_node.full_node: INFO     peer disconnected PeerInfo(_ip=IPv4Address('85.254.75.173'), _port=0)
2024-03-01T17:39:35.371 wallet wallet_server              : INFO     Cannot connect to host 78.42.236.162:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:36.150 wallet wallet_server              : INFO     Cannot connect to host 193.42.0.217:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:36.928 wallet wallet_server              : INFO     Cannot connect to host 104.48.169.185:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
2024-03-01T17:39:37.673 wallet wallet_server              : INFO     Cannot connect to host 51.183.134.230:8444 ssl:<ssl.SSLContext object at 0x000001F92E182C40> [The semaphore timeout period has expired]
